src/core/data/server.c

```diff
@@ -96,22 +97,47 @@ _server_pipe_write(void)
 }
 #endif
 
+static inline void
+_server_notify_worker(void)
+{
+#ifdef USE_EVENT_FD
```

Comment:
shall we model this function, style wise, after the read notification one, and simply include the code from the two write functions (eventfd, pipe) into this one? I think the LOC total is comparable.